docs/src/testing/gas-etimation.md

```diff
@@ -0,0 +1,34 @@
+# Gas estimation
+
+Transaction gas cost is always important for the users. Developers need to know if their contracts are well-optimised
+and gas-efficient.
+Forge supports gas estimation for each test case. All computations are based on the official
+[Starknet docs](https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/fee-mechanism/#how_much_fee_is_charged_high_level_overview).
+
+> ⚠️ In the next releases, gas estimation will be improved to include [onchain data cost](https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/fee-mechanism/#_on_chain_data)  ⚠️
```

Comment:
A task to remove this should be created or don't mention this in docs at all.

Reply:
I'd just remove it now

Reply:
I think this comment is important, because users can see that gas cost will change in the future